tests/foreman/ui/test_trend.py

```diff
@@ -0,0 +1,70 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Trend UI"""
+
+from fauxfactory import gen_string
+from robottelo.test import UITestCase
+from robottelo.ui.factory import make_trend
+from robottelo.ui.session import Session
+
+
+class TrendTest(UITestCase):
+    """Implements Trend tests in UI.
+
+    Please note that we have static amount of Trend types and trendables, so it
+    is not possible to generate values for them. In case you want to interact
+    with them through automation, it is must have condition to provide unique
+    elements from corresponding lists.
+    In most cases it will not be possible to re-run tests due condition of
+    uniqueness described above.
+
+    """
+
+    def test_create_trend_positive(self):
+        """@Test: Create new trend
+
+        @Feature: Trend - Positive Create
+
+        @Assert: Trend is created
+
+        """
+        name = 'Model'
+        with Session(self.browser) as session:
+            make_trend(session, trend_type=name)
+            search = self.trend.search(name)
+            self.assertIsNotNone(search)
+
+    def test_update_trend_positive(self):
+        """@Test: Update trend entity value
+
+        @Feature: Trend - Positive Update
+
+        @Assert: Trend entity is updated
+
+        """
+        name = gen_string('alpha')
```

Comment:
I would go with `utf8` because it have a more complex character definition and would raise error if multibyte strings are not accepted.